crates/viewer/re_view_dataframe/tests/basic.rs

```diff
@@ -46,9 +46,9 @@ pub fn test_unknown_timeline() {
 
     test_context.log_entity("some_entity".into(), |builder| {
         builder
-            .with_archetype(RowId::new(), [(timeline, 0)], &Scalar::new(10.0))
-            .with_archetype(RowId::new(), [(timeline, 1)], &Scalar::new(20.0))
-            .with_archetype(RowId::new(), [(timeline, 2)], &Scalar::new(30.0))
+            .with_archetype(RowId::new(), [(timeline, 0)], &Scalars::new([10.0]))
+            .with_archetype(RowId::new(), [(timeline, 1)], &Scalars::new([20.0]))
+            .with_archetype(RowId::new(), [(timeline, 2)], &Scalars::new([30.0]))
```

Comment:
should we maybe support/encourage this?

```suggestion
            .with_archetype(RowId::new(), [(timeline, 0)], &Scalars::from(10.0))
            .with_archetype(RowId::new(), [(timeline, 1)], &Scalars::from(20.0))
            .with_archetype(RowId::new(), [(timeline, 2)], &Scalars::from(30.0))
```